examples/tracing-http-propagator/src/server.rs

```diff
@@ -30,11 +41,11 @@ async fn handle_health_check(
     _req: Request<Incoming>,
 ) -> Result<Response<BoxBody<Bytes, hyper::Error>>, Infallible> {
     let tracer = get_tracer();
-    let mut span = tracer
+    let _ = tracer
```

Comment:
Doesn't this change means the span will be dropped immediately on this line? The span time will probably be off as it's no longer the execution time of the function

Reply:
Thanks! fixed.